index.js

```diff
@@ -1,80 +1,106 @@
 /*
-	MIT License http://www.opensource.org/licenses/mit-license.php
-	Author Tobias Koppers @sokra
+  MIT License http://www.opensource.org/licenses/mit-license.php
+  Author Tobias Koppers @sokra
 */
 var path = require("path");
 var loaderUtils = require("loader-utils");
 
 module.exports = function(content) {
-	this.cacheable && this.cacheable();
-	if(!this.emitFile) throw new Error("emitFile is required from module system");
+  this.cacheable && this.cacheable();
+  if(!this.emitFile) throw new Error("emitFile is required from module system");
 
-	var query = loaderUtils.getOptions(this) || {};
-	var configKey = query.config || "fileLoader";
-	var options = this.options[configKey] || {};
-	var config = {
-		publicPath: false,
-		useRelativePath: false,
-		name: "[hash].[ext]"
-	};
+  var query = loaderUtils.getOptions(this) || {};
+  var configKey = query.config || "fileLoader";
+  var options = this.options[configKey] || {};
```

Comment:
General Question: Is this still needed 😛  ? What does `this.options` contain ? 🙃